src/Label/ReadModels/Relations/Projector.php

```diff
@@ -83,6 +83,22 @@ public function applyLabelsImported(LabelsImportedEventInterface $labelsImported
         }
     }
 
+    public function applyLabelsReplaced(LabelsImportedEventInterface $labelsReplaced, Metadata $metadata): void
+    {
+        foreach ($labelsReplaced->getAllLabelNames() as $labelName) {
+            try {
+                $this->writeRepository->save(
+                    $labelName,
+                    $this->offerTypeResolver->getRelationTypeForReplaceLabel($labelsReplaced),
+                    $labelsReplaced->getItemId(),
+                    true
```

Comment:
I don't think we need the value of `true` here for the replace. In that case, we don't even need to implement this method because the `LabelAdded` and `LabelRemoved` will add the labels to the projection.